instance/models/mixins/storage.py

```diff
@@ -232,14 +232,23 @@ def get_s3_connection(self):
             self.s3_secret_access_key
         )
 
-    def _create_bucket(self, attempts=4, ongoing_attempt=1, retry_delay=4):
+    def _create_bucket(self, attempts=4, ongoing_attempt=1, retry_delay=4, location=None):
         """
         Create bucket, retry up to defined attempts if it fails
+        If you specify a location (e.g. 'EU', 'us-west-1'), it will use it.
```

Comment:
@lgp171188 Maybe mention what the method falls back to if caller doesn't specify `location`.